docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,679 @@
+---
+title: Introducing MUI Core v5.0
+description: After over 400 days of development and over 40 canary releases, we are excited to introduce MUI Core v5.0.0!
+date: 2021-09-15T00:00:00.000Z
+authors: ['oliviertassinari', 'eps1lon', 'mnajdova', 'michaldudak', 'siriwatknp']
+card: true
+---
+
+After over 400 days of development and over 40 canary releases, we are excited to introduce MUI Core v5.0.0!
+This release features some major highlights:
+
+- [High-level goals for v5](#high-level-goals-for-v5)
+- [A new brand](#a-new-brand)
+- [Improved customizability](#improved-customizability)
+  - [Migration from JSS to emotion](#migration-from-jss-to-emotion)
+  - [The sx prop](#the-sx-prop)
+  - [Dynamic props](#dynamic-props)
+  - [Global class names](#global-class-names)
+  - [Base components (alpha)](#base-components-alpha)
+- [Improved DX](#improved-dx)
+  - [Smaller demos in the docs](#smaller-demos-in-the-docs)
+  - [Props descriptions in IntelliSense](#props-descriptions-in-intellisense)
+  - [Migration from Enzyme to Testing Library](#migration-from-enzyme-to-testing-library)
+  - [TypeScript migration](#typescript-migration)
+  - [Strict Mode support](#strict-mode-support)
+- [A new product line: X](#a-new-product-line-x)
+  - [Data Grid](#data-grid)
+  - [Date Picker](#date-picker)
+- [New components](#new-components)
+  - [Improved Grid](#improved-grid)
+  - [More Material Design icons](#more-material-design-icons)
+  - [Stack](#stack)
+  - [Promotion from the lab](#promotion-from-the-lab)
+  - [New in the lab](#new-in-the-lab)
+- [v4 migration](#v4-migration)
+  - [Change of the package names](#change-of-the-package-names)
+  - [Change of the styling solution](#change-of-the-styling-solution)
+  - [Change on the supported platforms](#change-on-the-supported-platforms)
+- [Design kits](#design-kits)
+- [What's next?](#whats-next)
+  - [A public roadmap](#a-public-roadmap)
+  - [Unstyled components and hooks](#unstyled-components-and-hooks)
+  - [Second design system](#second-design-system)
+  - [MUI X](#mui-x)
```

Comment:
Is this section meant as a table of contents? If so, let's double check before merging that it's up-to-date

Reply:
Done and one issue found